docs/DeveloperGuide.adoc

```diff
@@ -14,10 +14,21 @@ ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
 
+== Introduction
+
+Welcome, fellow software developer! We are extremely delighted to see you. The CYNC community
+prides itself on being a friendly and helpful community for new developers.
+If you ever face any issue or uncertainty in the instructions, feel free to send an email
+to pohjie@cync.com so that we can sort out the issues for you and following contributors.
+
 == Setting up
 
 === Prerequisites
 
+Before you get started, please ensure that you have the following software and Java development
+kit installed on your computer. As downloading and setting up *IntelliJ* may take up some time,
+feel free to jump ahead and read through the instructions to get a rough idea of this project.
```

Comment:
I recognise that you are trying to make the guide more reader-friendly, but I feel that this part was kept short in the original DG for people who are coming in to quickly look for the pre-requisites instead of having to read through a chunk of text.

Reply:
Agreed regarding the poor phrasing for commits. As for the following comments, I see where you are coming from. We can take it both ways, we should probably come to consensus as a group which style we should adopt.